js-variables/variables.js

```diff
@@ -0,0 +1,28 @@
+let adminName = "Maryna";
+adminName = 10; 
+console.log(typeof adminName);
+
+let userName = "Maxim";
+userName = 20;
+console.log(typeof adminName);
+
+
+let title;
+title = "awersome";
+console.log(title);
+
+let clickButton;
+clickButton = "push";
```

Comment:
Same here 
```suggestion
let clickButton = "push";
```
you just need define and assing .